R/auciv.R

```diff
@@ -47,24 +53,8 @@ pk.calc.auxciv <- function(conc, time, c0, auxc, fun_auxc_last, ..., options = l
   auxc_final
 }
 
-#' Calculate AUC for intravenous dosing
-#'
-#' @details The AUC for intravenous (IV) dosing extrapolates the AUC back from
-#'   the first measurement to time 0 using c0 and the AUC calculated by another
-#'   method (for example the auclast).
-#'
-#' The calculation method takes the following steps:
-#'
-#' \itemize{
-#'   \item{`time = 0` must be present in the data with a measured concentration.}
-#'   \item{The AUC between `time = 0` and the next time point is calculated (`auc_first`).}
-#'   \item{The AUC between `time = 0` with `c0` and the next time point is calculated (`auc_second`).}
-#'   \item{The final AUC is the initial AUC plus the difference between the two AUCs (`auc_final <- auc + auc_second - auc_first`).}
-#' }
-#' @inheritParams pk.calc.auxciv
-#' @param auc The AUC calculated using `conc` and `time` without `c0` (it may be
-#'   calculated using any method)
-#' @return `pk.calc.auciv`: The AUC calculated using `c0`
+
+#' @describeIn pk.calc.auciv Calculate AUC for intravenous dosing with C0 back-extrapolation
```

Comment:
This needs to be `@describeIn pk.calc.auxciv` (adding the "x") so that it will go into that documentation.